tour-of-heroes/protractor-saucelabs.conf.js

```diff
@@ -0,0 +1,48 @@
+var defaultConfig = require('./protractor.conf.js').config;
+
+var testName = 'Tour of Heroes Smoke Tests'
+
+var ciConfig = {
+  // Note that seleniumAddress must be unset in order for SauceLabs to work.
+  seleniumAddress: false,
+  sauceUser: process.env.SAUCE_USERNAME,
+  sauceKey: process.env.SAUCE_ACCESS_KEY,
+
+  directConnect: false,
```

Comment:
Turns out the problem was I was using an old Sauce Labs account, which didn't have any VM minutes for automated testing. Instead of throwing an error, Sauce would just hang forever waiting to launch.

Also, this uncovered a bug in Protractor. The way 'beforeLaunch' is used to set up ts-node doesn't actually work with multiCapabilities, because of the way we fork in the launcher. I suspect that if you used test sharding, that would also break.